harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -134,18 +134,6 @@ def _set_data_loaders(self) -> None:
         nreplicas = self.context.distributed.size
```

Comment:
there's still a lot of updates missing in this file.  I would expect everything in `_metrics_utils` to be called from here, right?